### ISSUE TITLE
Using encoded HTML entities does not render correctly.

### ISSUE THREAD

**Issue by theomuller** [pveyes/htmr#10]
I am trying render some HTML content using your converter.

The content will look something like this: 
```'<p>Hej &amp;<br />↵&amp;<br />↵<br />↵&amp; &amp; &amp;</p>↵'```

When I use your converter it does not render the ampersands correctly. Instead it renders the encoded HTML entities as raw text.

I would bet that the problem occurs here:
```
const tempEl = document.createElement('div');
function escape(str) {
  tempEl.textContent = str;
  return tempEl.innerHTML;
}
```
Not sure if this will work but I'd suggest trying out using something like this instead: 
```
function decodeHtml(html) {
  const txt = document.createElement('textarea');
  txt.innerHTML = html;
  return txt.value;
}
```

**Comment by pveyes** [comment]
Hi @theomuller I've create a fix in #11, but I use `div` and `textContent` instead of `textarea` and `value`. I also added one test to confirm it's working, but feel free to propose another testcase that doesn't work with my solution

**Comment by pveyes** [comment]
Server side bundle entities decoding fixed in v0.4.4
